### ISSUE TITLE
Locale::getFallback returns false instead of null and infinite loop begins

### ISSUE THREAD

**Issue by karelVanGeerdeghom** [akeneo/pim-community-dev#9940]
I'm using Akeneo 2.3. It has hardcoded dependency on symfony 3.4.4.
With the latest version of the php-intl extension an infinte loop has been introduced in symfony core. (see https://github.com/symfony/symfony/issues/31106)

How can I override that Akeneo 2.3 has to have Symfony 3.4.4?

**Comment by karelVanGeerdeghom** [comment]
As a temporary fix I have solved it for now by changing my composer.json as such:
```
    "require": {
        "symfony/symfony": "3.4.24 as 3.4.4",
        "akeneo/pim-community-dev": "~2.3.31"
    },
```

**Comment by Doodoune** [comment]
Hi ! 

We upgrade Symfony version to 3.4.26 to fix Intl issues in the v2.3.41 
cf https://github.com/akeneo/pim-community-dev/blob/2.3/CHANGELOG-2.3.md#2341-2019-05-02

Please upgrade your PIM ;)

I close this issue, feel free to reopen it if needed.
Regards,
